wdfn-server/waterdata/templates/base.html

```diff
@@ -1,7 +1,11 @@
 {% extends 'base_plain.html' %}
 
 {% block page_css %}
-      <link rel="stylesheet" href="{{ 'main.css' | asset_url }}">
+    <link rel="stylesheet" href="{{ 'main.css' | asset_url }}">
+{% endblock %}
+
+{% block page_script %}
+    <script src="{{ 'scripts/uswds.min.js' | asset_url }}"></script>"
```

Comment:
If a page extends this template, it will automatically get the USWDS javascript. If this block is overriden, the page should be including the USWDS javascript in it's bundle/